setup.py

```diff
@@ -9,18 +9,21 @@
 
 python_requires = ">= 3.5"
 
-# MoSQITo dependancies
+# MoSQITo dependencies
 with open("requirements.txt", "r") as file:
     requirements = file.readlines()
     install_requires = "".join(
         requirements
     ).splitlines()  # remove endline in each element
 
-tests_require = ["pytest>=5.4.1", "pandas", "openpyxl"]
+tests_require = ["pytest>=5.4.1", "pandas", "openpyxl", "pytest","SciDataTool"]
 uff_require = [
-    "pyuff",
+    "pyuff"
 ]
-all_require = tests_require + uff_require
+scidatatool_require = [
+    "SciDataTool"
+]
+all_require = tests_require + uff_require + scidatatool_require 
```

Comment:
SciDataTool comes with tests_require and scidatatool_require. scidatatool_require can be omitted there

Reply:
I was thinking about people using mosqito with SDT but not wanting to check the tests... they don't need pytest, pandas and openpyxl in that case.